e2e/test_payloads_validator.py

```
@@ -0,0 +1,520 @@
+        return False
+
+    print("  ✅ Header PDF válido")
+
+    # Tentar ler (pode falhar ou ter avisos)
+    try:
+        reader = PdfReader(str(pdf_path))
+        text = ""
+        for page in reader.pages:
+            try:
+                text += page.extract_text()
+            except:
```

Ruff (0.14.5)

Do not use bare `except`

(E722)